govcd/adminorg.go

```diff
@@ -362,23 +367,58 @@ func (adminOrg *AdminOrg) removeAllOrgNetworks() error {
 
 // Forced removal of all organization catalogs
 func (adminOrg *AdminOrg) removeCatalogs() error {
-	for _, catalogs := range adminOrg.AdminOrg.Catalogs.Catalog {
-		// Get Catalog HREF
-		catalogHREF := adminOrg.client.VCDHREF
-		catalogHREF.Path += "/admin/catalog/" + strings.Split(catalogs.HREF, "/api/admin/catalog/")[1] //gets id
-		req := adminOrg.client.NewRequest(map[string]string{
-			"force":     "true",
-			"recursive": "true",
-		}, http.MethodDelete, catalogHREF, nil)
-		_, err := checkResp(adminOrg.client.Http.Do(req))
+	for _, catalog := range adminOrg.AdminOrg.Catalogs.Catalog {
```

Comment:
Could you fix the comment for this function. It's not valid godoc

Reply:
fixed